qiime2/core/archive/provenance_lib/_checksum_validator.py

```diff
@@ -144,7 +144,8 @@ def diff_checksums(zf: ZipFile) -> ChecksumDiff:
     exp = {}
     for line in zf.open(checksum_fp):
         fp, checksum = from_checksum_format(str(line, 'utf-8'))
-        exp[fp] = checksum
+        if pathlib.Path(fp).parts[0] != 'annotations':
+            exp[fp] = checksum
```

Comment:
```suggestion
        exp[fp] = checksum
```
Same here I think, we can trust the checksum file to not contain things we don't care about.

Reply:
~~The section above however (for `obs = checksum...`) does need some updated logic. But I probably haven't gotten to the function definition for it yet.~~  Found it!